data/data/ovirt/variables-ovirt.tf

```diff
@@ -19,6 +19,21 @@ variable "ovirt_password" {
   description = "The plain password of user to access Engine API"
 }
 
+variable "ovirt_cafile" {
```

Comment:
`ovirt_cafile` and `ovirt_ca_bundle` need a default since the fields in the go struct are `omitempty`.